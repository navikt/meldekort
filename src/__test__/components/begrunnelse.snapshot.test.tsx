import * as React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ProviderWrapper from '../testSetup/providerWrapper';
import BegrunnelseVisning from '../../app/components/meldekortdetaljer/begrunnelsevisning/begrunnelse';

Enzyme.configure({ adapter: new Adapter() });

it('BegrunnelseVelger', () => {
  const tree = shallow(
    <ProviderWrapper>
      <BegrunnelseVisning begrunnelse={'TEST'} />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
