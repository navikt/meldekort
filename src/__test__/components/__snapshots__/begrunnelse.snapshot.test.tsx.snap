// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BegrunnelseVelger 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <Connect(IntlProvider)
    defaultLocale="nb"
    locale="nb"
  >
    <BegrunnelseVisning
      begrunnelse="TEST"
    />
  </Connect(IntlProvider)>
</Provider>
`;
